### PR TITLE
New Rule: BEC/Fraud: PenPal Scam

### PR DESCRIPTION
**Pull request by morriscode**

### CODE REVIEW

**Comment by morriscode** [comment]
This is super specific hunts return the only know bad and no fps

File: detection-rules/bec_fraud_penpal_scam.yml [hunk 1]
```diff
@@ -0,0 +1,57 @@
+name: "BEC/Fraud: PenPal Scam"
```
Comment (jkamdjou) [inline detection-rules/bec_fraud_penpal_scam.yml:1]:
```suggestion
name: "PenPal Scam"
```

**Review by jkamdjou — APPROVED** [review]
ideas in case you want to consider adding, defer to you:
- NLU entity
- no links or attachments

**Comment by bfilar** [comment]
@morriscode perhaps not required, but a:

```
any(ml.nlu_classifier(body.current_thread.text).entities, .name == "request")
```

may work as well

**Pull request merged**